src/sce-elf-defs.h

```diff
@@ -91,6 +91,21 @@ typedef struct SCE_TYPE(sce_module_imports_short) {
 	SCE_PTR(const void **) var_entry_table;		/* Pointer to array of data pointers to write to */
 } SCE_TYPE(sce_module_imports_short);
 
+typedef struct SCE_TYPE(sce_process_param_old) {
+	uint32_t size;                          /* 0x30 */
+	uint32_t magic;                         /* PSP2 */
+	uint32_t version;                       /* Unknown, but it could be 6 */
```

Comment:
version to be 5. right?

how about rename types to `sce_process_param_v5` and `sce_process_param_v6`?
it might be a bad name too.

Reply:
done